data/plugins/language_lua.lua

```diff
@@ -27,9 +27,53 @@ syntax.add {
     { pattern = "%.%.%.?",                   type = "operator" },
     { pattern = "[<>~=]=",                   type = "operator" },
     { pattern = "[%+%-=/%*%^%%#<>]",         type = "operator" },
-    { pattern = "[%a_][%w_]*()%s*%f[(\"'{]", type = {"function", "normal"} },
-    { pattern = "[%a_][%w_]*",               type = "symbol" },
+    -- Metamethods
+    { pattern = "__[%a][%w_]+",              type = "function" },
+    { pattern = "%.()__[%a][%w_]+",
```

Comment:
Maybe a `*` instead of `+` could be better.

Reply:
are there metamethods with a single character? or to match custom metamethods?

Reply:
Yeah, I don't think there's a reason to limit it to two or more characters.